server/app/cloud.go

```diff
@@ -4,94 +4,18 @@
 package app
 
 import (
-	"errors"
-	"fmt"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
 
-	"github.com/mattermost/mattermost-server/v6/shared/mlog"
-
-	mmModel "github.com/mattermost/mattermost-server/v6/model"
-
-	"github.com/mattermost/focalboard/server/model"
-	"github.com/mattermost/focalboard/server/utils"
+	mmModel "github.com/mattermost/mattermost/server/public/model"
 )
 
-var ErrNilPluginAPI = errors.New("server not running in plugin mode")
-
-// GetBoardsCloudLimits returns the limits of the server, and an empty
-// limits struct if there are no limits set.
-func (a *App) GetBoardsCloudLimits() (*model.BoardsCloudLimits, error) {
-	// ToDo: Cloud Limits have been disabled by design. We should
-	// revisit the decision and update the related code accordingly
-	/*
-		if !a.IsCloud() {
-			return &model.BoardsCloudLimits{}, nil
-		}
-
-		productLimits, err := a.store.GetCloudLimits()
-		if err != nil {
-			return nil, err
-		}
-
-		usedCards, err := a.store.GetUsedCardsCount()
-		if err != nil {
-			return nil, err
-		}
-
-		cardLimitTimestamp, err := a.store.GetCardLimitTimestamp()
-		if err != nil {
-			return nil, err
-		}
-
-		boardsCloudLimits := &model.BoardsCloudLimits{
-			UsedCards:          usedCards,
-			CardLimitTimestamp: cardLimitTimestamp,
-		}
-		if productLimits != nil && productLimits.Boards != nil {
-			if productLimits.Boards.Cards != nil {
-				boardsCloudLimits.Cards = *productLimits.Boards.Cards
-			}
-			if productLimits.Boards.Views != nil {
-				boardsCloudLimits.Views = *productLimits.Boards.Views
-			}
-		}
-
-		return boardsCloudLimits, nil
-	*/
-
-	return &model.BoardsCloudLimits{}, nil
-}
-
-func (a *App) GetUsedCardsCount() (int, error) {
-	return a.store.GetUsedCardsCount()
-}
-
-// IsCloud returns true if the server is running as a plugin in a
-// cloud licensed server.
-func (a *App) IsCloud() bool {
-	return utils.IsCloudLicense(a.store.GetLicense())
-}
-
-// IsCloudLimited returns true if the server is running in cloud mode
-// and the card limit has been set.
-func (a *App) IsCloudLimited() bool {
-	// ToDo: Cloud Limits have been disabled by design. We should
-	// revisit the decision and update the related code accordingly
-
-	// return a.CardLimit() != 0 && a.IsCloud()
-
-	return false
-}
-
 // SetCloudLimits sets the limits of the server.
 func (a *App) SetCloudLimits(limits *mmModel.ProductLimits) error {
 	oldCardLimit := a.CardLimit()
 
 	// if the limit object doesn't come complete, we assume limits are
 	// being disabled
 	cardLimit := 0
```

Comment:
There is no need to set or handle any cloud limits, we can remove this whole file

Reply:
The only function in use from this file is `GetUsedCardsCount`, but I don't think this function belongs here so I will be moving the function to the `statistics.go` file. And will delete `cloud.go`.